Excel-custom-functions/Batching/samplewithtodos.js

```diff
@@ -0,0 +1,76 @@
+//Custom Functions Batching Sample with TODO items
+
+//TODO #1: Create custom functions for adding and multiplying numbers.
```

Comment:
Remove gerunds whenever you can. Including code samples! "Create custom functions that perform addition and multiplication calculations" -or- "Create custom functions to add and multiply numbers."